.prettierignore

```diff
@@ -10,3 +10,4 @@ e2e/tmp
 public/build/
 public/sass/*.generated.scss
 devenv/
+public/lib
```

Comment:
I think this might need to be `public/lib/monaco` as it looks like there's some code in `public/lib/monaco-languages` that is our own. I think @joshhunt has a better idea of what's going on in this folder.

Reply:
+1 I can confirm this.